sql/core/src/test/scala/org/apache/spark/sql/execution/SQLFunctionSuite.scala

```diff
@@ -74,4 +74,17 @@ class SQLFunctionSuite extends QueryTest with SharedSparkSession {
           |""".stripMargin), Seq(Row(2), Row(4)))
     }
   }
+
+  test("SQL scalar function with default value") {
```

Comment:
can we add another test with SQL table functions?

Reply:
This test is for convenience when debugging. The SQL tests already check that it works with table functions.